main.go

```diff
@@ -237,13 +240,13 @@ func runAuthorizationServer(cmd *cobra.Command, _ []string) {
 
 	baseManagerOptions := ctrl.Options{
 		Scheme:                 scheme,
-		Port:                   opts.webhookServicePort,
-		MetricsBindAddress:     opts.metricsAddr,
+		WebhookServer:          webhook.NewServer(webhook.Options{Port: opts.webhookServicePort}),
+		Metrics:                metricsserver.Options{BindAddress: opts.metricsAddr},
 		HealthProbeBindAddress: opts.healthProbeAddr,
 		LeaderElection:         false,
 	}
 	if opts.watchNamespace != "" {
-		baseManagerOptions.Namespace = opts.watchNamespace
+		baseManagerOptions.Cache.DefaultNamespaces = map[string]cache.Config{opts.watchNamespace: {}}
```

Comment:
Interesting. So now we can supporting watching selected namespaces (plural)?

We need to test this with Authorino deployed in both [modes](https://docs.kuadrant.io/authorino/docs/architecture/#cluster-wide-vs-namespaced-instances) (i.e. cluster-wide and namespaced.)

Reply:
> Interesting. So now we can supporting watching selected namespaces (plural)?

Yeah, it seems the BaseOptions has moved the `Namespace` option to `Cache.Options.DefaultNamespaces` 🤔  - https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/cache#Options

By default it will watch from all namespaces unless defined in this slice

- https://github.com/kubernetes-sigs/controller-runtime/blob/v0.16.3/pkg/manager/manager.go#L109-L111

So it seems possible to support multi namespace via this if we so choose at some point 🤔 

> We need to test this with Authorino deployed in both [modes](https://docs.kuadrant.io/authorino/docs/architecture/#cluster-wide-vs-namespaced-instances) (i.e. cluster-wide and namespaced.

Agreed 👍

Reply:
> BaseOptions has moved the `Namespace` option to `Cache.Options.DefaultNamespaces`

So it seems. It's in the release notes of [v0.15.0](https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.15.0) (since v0.14.5):

> `cache.Options.Namespace` has been removed in favor of `cache.Options.Namespaces`, a slice.

And I think that changed again later, in [v0.16](https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.16.0), with:

- https://github.com/kubernetes-sigs/controller-runtime/pull/2421